src/boxes/file_type.rs

```diff
@@ -0,0 +1,54 @@
+/*
+ * © 2022 Arastoo Bozorgi
+ * All rights reserved.
+ */
+
+use crate::boxes::Mp4Box;
+use crate::Error;
+use crate::Header;
+
+#[derive(Debug)]
+pub struct FileType {
+    header: Header,
+    data: Vec<u8>,
```

Comment:
warning: field is never read: `data`
  --> src\boxes\file_type.rs:13:5
   |
13 |     data: Vec<u8>,
   |     ^^^^^^^^^^^^^
   |
   = note: `#[warn(dead_code)]` on by default